condor_watch_q.py

```diff
@@ -513,8 +522,16 @@ def row_data_from_job_state(clusters):
 
     row_data[TOTAL] = sum(row_data.values())
     row_data[ACTIVE_JOBS] = ", ".join(active_job_ids)
-
-    return row_data
+    
+    summary["TOTAL"] += row_data[TOTAL]
```

Comment:
Let's use the predefined string constants for these (including the ones in the `JobStatus` enum, where appropriate).